specs/market-api.yaml

```diff
@@ -940,22 +949,22 @@ components:
         - properties
         - constraints
       properties:
+        timestamp:
+          type: string
+          format: date-time
```

Comment:
this should be moved to Offer/Demand/Proposal as it is not defined via caller, but via the service

Reply:
or maybe even to AgreementProposal (as optional) and Agreement (as required)

Reply:
Agreed (apart from AgreementProposal - if it is the server which assigns this, and AgreementProposal is requestBody structure...)